toDoApp/database/migrations/2018_11_07_122822_create_cards_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+
+class CreateCardsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('cards', function (Blueprint $table) {
+            $table->increments('id');
+            $table->timestamps();
+            $table->string('title');
+            $table->string('content');
+            $table->integer('user_id');
```

Comment:
Ovo je foreign key, dodaj to
https://laravel.com/docs/5.7/migrations#foreign-key-constraints